mmv1/templates/terraform/env_var_context.go.tmpl

```diff
@@ -30,4 +30,7 @@
 		"{{$varKey}}": envvar.GetTestZoneFromEnv(),
 		{{- end }}
 	{{- end }}
+	{{- if and $.HasNewLine (gt (len $.TestEnvVars) 0)}}
```

Comment:
> $.HasNewLine

I didn't realize we had this lol

Reply:
It is a passed when calling the EnvVarContext templates.